docs/volume-scheduling.md

```diff
@@ -80,6 +80,8 @@ $ kubectl directpv label drives --drives=nvme1n1 tier=fast
 $ create-storage-class.sh fast-tier-storage 'directpv.min.io/tier: fast'
 ```
 
+p.s. the label keyName has to be prefixed with `directpv.min.io/`, otherwise it won't be considered
```

Comment:
`label` command clearly outputs what label is set. It is user's responsibility to use them correctly than guess work.
```
[root@master ~]# ./kubectl-directpv_4.0.8_linux_amd64 label drive --drives=vdb tier=fast
Label 'directpv.min.io/tier:fast' successfully set on master/vdb
```

However I will add a check to `create-storage-class.sh` to error out if `directpv.min.io/` prefix is missed out.